lxd/db/cluster/open.go

```diff
@@ -211,6 +212,15 @@ INSERT INTO nodes(id, name, address, schema, api_extensions, arch, description)
 				return err
 			}
 
+			// Server admin group
+			var adminGroupStmt strings.Builder
+			adminGroupStmt.WriteString("INSERT INTO auth_groups (name, description) VALUES ('admin', 'Server admin');")
+			adminGroupStmt.WriteString(fmt.Sprintf("INSERT INTO auth_groups_permissions (auth_group_id, entity_type, entity_id, entitlement) VALUES (1, %d, 0, '%s');", entityTypeServer{}.code(), string(auth.EntitlementAdmin)))
```

Comment:
Would you mind using query params like https://github.com/canonical/lxd/pull/15215/commits/3b901b2e3fbe4ef4e30d5eaeec0c2b33769584ca#diff-ef6346dbfa30f559694e6b5ce2146257339bb50f12b697fadc82614bf0b49af4R194 i reaslise its not strictly neccesary for security, but would avoid need for fmt.Sprintf.